templates/data-sources/host.md.tmpl

```diff
@@ -19,19 +19,23 @@ linkTitle: "{{.Name}}"
 page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
 subcategory: ""
 description: |-
-{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+
+This datasource is used to query the existing host from PowerStore array. The information fetched from this datasource can be used for getting the details for further processing in resource block.
```

Comment:
@Sakshi-dell I am little confused why tmpl is updated and not the schema description. We should keep this in source code and it will be picked up by documentation generation.

Reply:
Updated